website/templates/log_templates.mako

```diff
@@ -191,6 +191,19 @@ invisible on
   {{/ifnot}}
 </script>
 
+<script type="text/html" id="addon_file_renamed">
+  {{#if params.source.materialized.endsWith('/')}}
```

Comment:
@chennan47 True, I was just following the format of the other scripts (i.e. addon_file_moved) they have duplicate code too. Do you want me to change those as well?